e2e-tests/steps/offenceAndLicenceInformationSection.ts

```diff
@@ -3,15 +3,38 @@ import { ApplyPage, TaskListPage } from '../pages/apply'
 
 export const completeCommunitySupervisionAndCurrentOffencesTask = async (page: Page, name: string) => {
   const taskListPage = new TaskListPage(page)
-  await taskListPage.clickTask('Community supervision and current offences')
+  await taskListPage.clickTask('Add alleged offences')
 
+  await completeAllegedOffenceDetailsPage(page, name)
+  await completeAllegedOffencesPage(page, name)
```

Comment:
should alleged offences not sit within their own task function definition?

Reply:
Yes!